### PR TITLE
Update doc: add known incompatible tools

### LINKED ISSUE #81: Cannot find package.json in path
Hi,

I'm using module-alias with ncc compiler (https://github.com/zeit/ncc) and seeing weird behavior. If I run the code without the ncc compiler, then it's working fine. However, after ncc compiler, then module-alias is throwing error that it cannot find package.json at some path, even though the path does contain package.json file. 

I tried with passing a static path for the module-alias init, and still getting the same error.

After debugging and trying to modify the code of module-alias, I find that if I were to add a temporary variable to join the path first, before the npmPackage require being called, then things will work. 

line 171
```
for (var i in candidatePackagePaths) {
    try {
      base = candidatePackagePaths[i]

      var packageJsonPath = nodePath.join(base, 'package.json')
      npmPackage = require(packageJsonPath)
      break
    } catch (e) {
      // noop
    }
  }
```

I'm not really sure why assigning the path to a temporary variable before `require` fixes the problem. Any idea?

### PR DESCRIPTION
**Pull request by Kehrlann**
- Closes #81

### CODE REVIEW

**Comment by Kehrlann** [comment]
Pinging @ilearnio , if you have a minute to review this.

**Comment by ilearnio** [comment]
Hey @Kehrlann. Sorry for delays, I'll try to review things today the evening.

File: README.md [hunk 1]
```diff
@@ -143,6 +143,16 @@ module.exports = {
...
 ```
 
+More details on the [official documentation](https://webpack.js.org/configuration/resolve).
+
+## Known incompatibilities
+
+This module does not play well with:
+
+- [Jest](https://jestjs.io), which discards node's module system entirely to use it's own module system, bypassing module-alias.
+- All kinds of front-end frameworks (React, Vue, ...), as they tend to use Webpack. Use Webpack's [resolve.alias](https://webpack.js.org/configuration/resolve/#resolvealias) mechanism instead.
```
Comment (ilearnio) [inline README.md]:
```suggestion
- Front-end JavaScript code. Do not expect it to work with frameworks like (React, Vue, ...) as they tend to use Webpack. Use Webpack's [resolve.alias](https://webpack.js.org/configuration/resolve/#resolvealias) mechanism instead.
```

Comment (ilearnio) [reply README.md]: Also maybe this line should be first

Comment (Kehrlann) [reply README.md]: Great idea. Thanks !

**Comment by ilearnio** [comment]
Awesome stuff! Merged

**Pull request merged**